pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -246,6 +261,17 @@ func (a *ApplicationBackupController) createBackupLocationPath(backup *stork_api
 	return nil
 }
 
+func createVMRuleCr(rule *stork_api.Rule) error {
+	_, err := storkops.Instance().GetRule(rule.Name, rule.Namespace)
+	if err != nil {
+		_, err = storkops.Instance().CreateRule(rule)
+		if err != nil {
```

Comment:
Better to add AlreadyExists error check and ignore the failure. Some time update might fail and we might re-enter again

Reply:
Added check for not found